src/Core/Cart/Calculator.php

```diff
@@ -186,7 +186,11 @@ public function getRowTotalWithoutDiscount()
     {
         $amount = new AmountImmutable();
         foreach ($this->cartRows as $cartRow) {
-            $amount = $amount->add($cartRow->getInitialTotalPrice());
+            if ($cartRow->getRoundType() !== $cartRow::ROUND_MODE_ITEM) {
```

Comment:
Is there a test (behat or not) to validate this behaviour and prevent future regression?
This can be done in another issue/PR I think, but it's important nonetheless

Reply:
Well my fix only applies to the round mode item, for which I added a scenario. 

In other cases the behavior remains ISO, and is already tested in the behat tests dedicated to rounding types.